pkgs/development/python-modules/unsloth/default.nix

```diff
@@ -112,19 +127,22 @@ buildPythonPackage rec {
       cudaPackages312 = cudaPackages.overrideScope (final: prev: {
         python3Packages = python.pkgs;
```

Comment:
I forgot how to handle splicing correctly here, but I'd guess you just ask `callPackage` for `pythonPackages` as an argument